pydatastructs/strings/tests/test_suffix_tree.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from pydatastructs.strings.suffix_tree import SuffixTree
+
+
+class SuffixTreeTest(unittest.TestCase):
```

Comment:
We don't use `unittest.TestCase` (or `unittest` at all). Please follow the testing pattern as in https://github.com/codezonediitj/pydatastructs/blob/main/pydatastructs/strings/tests/test_trie.py.